BookingsApi/BookingsApi.Domain/JudiciaryParticipant.cs

```diff
@@ -14,7 +14,7 @@ public JudiciaryParticipant(string displayName, JudiciaryPerson judiciaryPerson,
             UpdateDisplayName(displayName);
             JudiciaryPerson = judiciaryPerson;
             UpdateHearingRoleCode(hearingRoleCode);
-            ContactEmail = String.IsNullOrWhiteSpace(optionalContactEmail) ? null : optionalContactEmail;
+            ContactEmail = String.IsNullOrWhiteSpace(optionalContactEmail) ? judiciaryPerson.Email : optionalContactEmail;
```

Comment:
We should use the the GetEmail accessor method, and set this back to the way it was (see below)

Reply:
done